src/commands/wait_for_workflow.yml

```diff
@@ -133,17 +136,33 @@ steps:
 
         current_wait_time=0
 
+        WORKFLOW_STATE_FAIL="(not_run|failed|error|failing|on_hold|canceled|unauthorized)"
+
         while true; do
           get_num_running_workflows
+          wf_output_file=/tmp/swissknife/"wf_${CIRCLE_WORKFLOW_ID}"_finished.json
+
           if [[ $total_workflows_running == "0" || $can_i_run == "true" ]]; then
```

Comment:
Should we do double quotes everywhere in comprasion? 

`if [[ $finished_wf_status =~ ${WORKFLOW_STATE_FAIL} ]]; then`

`if [[ "${finished_wf_status}" =~ "${WORKFLOW_STATE_FAIL}" ]]; then`

etc